Storage/SQL/BrowserTable.swift

```diff
@@ -166,9 +166,9 @@ public class BrowserTable: Table {
         let status = SyncStatus.New.rawValue
 
         let localArgs: Args = [
-            BookmarkRoots.RootID,    BookmarkRoots.RootGUID,          type, BookmarkRoots.RootGUID, status, now,
```

Comment:
This was a case where spaces were added to make things cleaner in the editor by allowing things to align. I'm usually not a fan of these.